sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -474,6 +479,52 @@ case class Lag(input: Expression, offset: Expression, default: Expression)
   override val direction = Descending
 }
 
+/**
+ * The NthValue function returns the value of `input` at the row that is the `offset`th row of
+ * the window frame (counting from 1). Offsets start at 0, which is the current row. When the
+ * value of `input` is null at the `offset`th row or there is no such an `offset`th row, null
+ * is returned.
+ *
+ * @param input expression to evaluate `offset`th row of the window frame.
+ * @param offset rows to jump ahead in the partition.
+ */
+@ExpressionDescription(
+  usage = """
+    _FUNC_(input[, offset]) - Returns the value of `input` at the row that is the`offset`th row
```

Comment:
nit: add a space between `the` and `offset`

Reply:
OK